web-frontend/.env.standalone

```diff
@@ -1,2 +1,4 @@
-VITE_BASE_PATH="/web-app/"
-VITE_API_URL="/api/"
\ No newline at end of file
+VITE_BASE_APP="/web-app"
+VITE_BASE_API="/api"
+# VITE_ENV="dev"
+VITE_ENV="prod"
```

Comment:
so which one it is ? dev or prod ? :p

Reply:
you choose haha, in dev mod, mirage will catch your API calls. I don't know yet how to use it properly, we will see.